src/main/webapp/script.js

```diff
@@ -83,6 +90,19 @@ async function fetchForum() {
   });
 }
 
+/**
+ * Fetches notifications of the signed in user.
+ */
+function fetchNotifications() {
+  fetch('/notification').then(response => response.json()).then((notificationsJson) => {
+    const notificationsElement = document.getElementById('inbox-dropdown');
+    notificationsElement.innerHTML = '';
+    for (const notification of notificationsJson) {
+      notificationsElement.appendChild(createNotificationsElement(notification));
+    }
+  });
+}
+
```

Comment:
This seems unrelated to mentor verification. Please try to separate changes like these from each other in future PRs!

Reply:
Sorry, I think this was caused by merge conflicts.